app/views/spree/order_mailer/cancel_email.html.haml

```diff
@@ -1,11 +1,30 @@
-%h3
-  = t(".customer_greeting", name: @order.bill_address.firstname)
-%h4
-  = t(".instructions")
-%span.clear
+%table.social.white-bg{:width => "100%"}
+  %table.column{:align => "left"}
+    %tr
+      %td
+        %h3
+          = t(".customer_greeting", name: @order.bill_address.firstname)
+        %h4
+          = t(".instructions_html", distributor: @order.distributor.name )
+        %img.float-left{:src => "#{@order.distributor.logo.url(:medium)}"}/
+
+
+%p.callout
+  = t(".dont_cancel", email: @order.distributor.contact.email)
 
 %p &nbsp;
```

Comment:
this usually looks like  %p=string
I wonder if there is any difference.

Reply:
A paragraph with an empty space in it? Maybe a line break ( `%br`) would be more appropriate here instead?

Reply:
@Matt-Yorkley are you  talking about `%pv &nbsp`?

Reply:
It looks very strange to me, but I guess it was here before... :+1:

Reply:
Yes, it was, so I just left it.